lib/pubsub/index.js

```diff
@@ -13,8 +16,14 @@ function PubSub(options) {
   // isn't complete until the callback returns from Redis
   // Maps channel -> true
   this.subscribed = {};
+
+  var self = this;
+  this._defaultCallback = function(err) {
```

Comment:
I'm impressed you repeated this pattern from QueryEmitter! Great job. 🎉 

For future people reading this PR, the pattern means that we only have to create this callback function and the closure of the `pubSub` reference one time. The alternative would have been to create an anonymous function every time we publish. This is an ideal application of the same optimization, and it is a good use of consistency in naming and code patterns.